ChainEval/configs/bbh/gemma2b-it/eval.yaml

```diff
@@ -15,9 +15,8 @@ metrics:
       - "\\.$"
 
 filter_list:
-    - name: "get-answer"
-      filter:
-        - function: "regex"
-          # regex_pattern: "The answer is (\\-?[0-9\\.\\,]+)."
-          regex_pattern: "The answer is (.*)."
-        - function: "take_first"
\ No newline at end of file
+  - name: "get-answer"
+    filter:
+      - function: "regex"
+        regex_pattern: "(?<=the answer is )(.*)(?=.)"
```

Comment:
@Anshumaan-Chauhan02 This regex looks different to other regex.
`regex_pattern: "The answer is (.*)."` Any reason?

Reply:
It looks like it is slightly different to the configs we used in gsm8k.

Reply:
Yes, because the final answer format for both of them is different, BBH also consists MCQs and other types of questions (unlike GSM8K)

Reply:
Okay! Resolving this